src/contracts/core/DelegationManager.sol

```diff
@@ -358,8 +389,11 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
                 // create the new storage
                 bytes32 newRoot = calculateWithdrawalRoot(migratedWithdrawal);
                 // safety check to ensure that root doesn't exist already -- this should *never* be hit
-                require(!pendingWithdrawals[newRoot], "DelegationManager.migrateQueuedWithdrawals: withdrawal already exists");
-                pendingWithdrawals[newRoot] = true;
+                require(!pendingWithdrawals(newRoot), "DelegationManager.migrateQueuedWithdrawals: withdrawal already exists");
```

Comment:
there are no unmigrated withdrawals (pending review), so ideally we can delete this after review

Reply:
yes, I'd like to delete but there's a bunch of tests and interface stuff so I didn't bother.
we should get rid of it though; that'll help with contract code size for sure.